src/storage/storage-async.ts

```diff
@@ -414,8 +452,9 @@ export class StorageAsync implements IStorageAsync {
         return ingestEvent;
     }
 
-    // overwrite every doc with an empty one, from this author:
-    // return the number of docs changed, or -1 if error.
```

Comment:
We need to tell end-users about the return value so they know if an error happened

Reply:
Yes, updated.